content/community/_index.md

```diff
@@ -1,64 +1,55 @@
 ---
 title: Community
-menu: none
+menu: main
 weight: 20
 ---
 
 👋 Welcome to the Keptn community!
 
 This is the starting point for joining and contributing to the Keptn community - improving docs, improving code, discussing new features and use cases etc. For all collaboration and communication going on in all keptn projects, please read and follow our [Code of Conduct](https://github.com/keptn/community/blob/master/CODE_OF_CONDUCT.md).
 
+## Slack 
 
-## Communication 
-
-### Slack 📟
-
-We have a dedicated [Keptn slack workspace](https://slack.keptn.sh) (for in-depth discussions and getting in touch with the team) as well as a #keptn channel in the CNCF Slack workspace. We are happy to talk to you!
+We have a dedicated [Keptn slack workspace](https://slack.keptn.sh) (for in-depth discussions and getting in touch with the team) as well as a #keptn channel in the [CNCF Slack workspace](http://slack.cncf.io/). We are happy to talk to you!
 
 ✅ Chat with other people about using Keptn
```

Comment:
Can you please change the icons to :heavy_check_mark: ` : heavy_check_mark : `

Reply:
we agreed to keep it as the heavy check mark is black on MacOS and not green